cmd/kube-apiserver/app/server.go

```diff
@@ -120,8 +119,7 @@ func NewAPIServer() *APIServer {
 		SecurePort:             6443,
 		APIRate:                10.0,
 		APIBurst:               200,
-		APIPrefix:              "/api",
-		ExpAPIPrefix:           "/experimental",
+		APIPrefix:              "/kube-api",
```

Comment:
We shouldn't change the meaning of the existing APIPrefix flag. We should create a new flag that would apply to new groups.